app/javascript/components/patient/assessment/AssessmentTable.js

```diff
@@ -303,6 +303,17 @@ class AssessmentTable extends React.Component {
       </Dropdown>
     );
   };
+  /**
+   * Adds an asterisk to the reporter if the assessment has been edited.
+   * @param {Object} data - Data about the cell the filter is called on
+   */
+  createReporterLabel = data => {
+    const rowData = data.rowData;
+    if (rowData.who_reported != 'Monitoree') {
+      return <div>{rowData.who_reported + '*'}</div>;
+    }
+    return <div>{rowData.who_reported}</div>;
```

Comment:
The original reporter for an assessment isn't always "Monitoree"- it can be a user. So the logic here will not be sufficient. But we also don't have a great way to fix this, since we don't currently track the original reporter in the database. I went ahead and asked about this in Slack [here](https://mitrecorp.slack.com/archives/C01A6P94R9B/p1627567291010000?thread_ts=1626364424.026600&cid=C01A6P94R9B).

So for now, please just remove this, and I am creating a separate task for it.